action.yml

```diff
@@ -72,6 +72,9 @@ inputs:
   prerelease-prefix:
     description: 'A prefix to apply to the prerelease version number'
     required: false
+  base-tag:
+    description: 'Specifies the starting release base tag and the changelog will include all the changes made in releases that occur after the base release, up until the latest release.'
```

Comment:
this description is a bit confusing. what the base-tag should do is use that instead of the latest as a base, but everything else is unchanged (I'm referring to the "up until the latest release" part)